### PR TITLE
Ignore version.ts when checking code style

### PR DESCRIPTION
**Pull request by ltrung**
**Issue #:** 
`npm run check` which check codestyles will flag the auto-generated file `version.ts` for copyright violation.

**Description of changes:**
Add a function to skip git ignore file when running copyright check. This is the same function in JS SDK.

**Testing**
1. Have you successfully run `npm run build:release` locally? Yes

2. How did you test these changes? Manually run `npm run check`

3. If you made changes to the component library, have you provided corresponding documentation changes? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/check-codestyle.js [hunk 1]
```diff
@@ -11,6 +11,23 @@ const { logger, process } = require('./utilities');
...
 let exitCode = 0;
 
+const ignoreMemo = {};
+const isIgnored = (file) => {
+  if (file in ignoreMemo) {
+    return ignoreMemo[file];
+  }
+  try {
+    // If this returns zero, it means the file is ignored by git; skip it.
+    exec(`git check-ignore -q '${file}'`);
```
Comment (devalevenkatesh) [inline scripts/check-codestyle.js:21]: Returns zero as in fails and we go in catch block? Trying to understand if we have to check the console logged value and the decide.

Comment (ltrung) [reply scripts/check-codestyle.js:21]: 0 in bash is actually passing :)

Comment (devalevenkatesh) [reply scripts/check-codestyle.js:21]: Ok thanks, so when the status exits with 1 its an error and its catched, thanks for explanation 🙏

**Review by devalevenkatesh — APPROVED** [review]

File: scripts/check-codestyle.js [hunk 2]
```diff
@@ -83,10 +99,8 @@ tests().forEach(file => {
 
 const spdx = '// SPDX-License-Identifier: Apache-2.0';
 
-let allYears = [];
-
 allFiles().forEach(file => {
-  if (file.endsWith('.d.ts') || file.includes('.DS_Store')) {
+  if (file.endsWith('.d.ts') || isIgnored(file)) {
```
Comment (devalevenkatesh) [inline scripts/check-codestyle.js:103]: May be move this or check to `isIgnored` now?

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**